docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -785,10 +785,27 @@ Any entries yielded this way will be attached to the `Handled Output` event for
 
 Additionally, if the handled output is part of a software-defined asset, these metadata entries will also be attached to the materialization event created for that asset and show up on the Asset Details page for the asset.
 
+## Full list of built-in I/O managers
```

Comment:
```suggestion
## Built-in I/O managers
```

I don't think we need `Full list of` - this is much shorter and gets the point across